style.css

```diff
@@ -134,3 +147,225 @@ body {
 .social-media ul li a {
   color: #505f79;
 }
+
+.works-section {
+  display: flex;
```

Comment:
- [ ] As stated in the project requirements,
 ### In order to lay out the elements in "Works" section you should use CSS Grid and Flexbox combined (it is mandatory that you use both). Use CSS Grid in the main container that contains all of the cards and use Flexbox to lay out an individual card.

Please use `display: grid;` to layout the elements in this section.😊

Reply:
- [ ] [REQUIRED] Kindly implement this, the container of your projects must have a `display: grid;`